### ISSUE TITLE
second deck starts fetching the Media files of the first one

### ISSUE THREAD

**Issue by hssm** [ankidroid/Anki-Android#1477]
Originally reported on Google Code with ID 574

```
By Tom:

As I saw the support for dropbox,  I jumped and updated my current
release to AnkiDroid 0.6 beta 13,
I was able to download the Media from dropbox to SD card by Download
missing media menu on the Deck popup.

However the issue came when I synced my second deck, It started
fetching the Media files of the older one(The one it successfully
fetched in the start) and obviously it did not find thouse in my
second Deck media directory of dropbox and throws error
"Error with file: filename.mp3"

In summary, The first decks Download missing media works
When the next decks missing media is pulled, some how Ankidroid keep
reference of the First deck files and thus fails.

Not sure If I am clear. Please feel free to ask.
```

Reported by `nicolas.raoul` on 2011-04-18 11:12:57
